.github/issue_labeler.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# PR template regexp's for issue labeler
+bug:
+    - '\[x\]\sbug\sfix'
+enhancement:
+    - '\[x\]\snew\sfeature'
+tests:
+    - '\[x\]\stests'
+infrastructure:
+    - '\[x\]\sinfrastructure'
+documentation:
+    - '\[x\]\sdocumentation'
```

Comment:
These kind of template regexps are applicable only for pull requests since we have a PR template with such entries but.

Reply:
As mentioned below, I can create an issue template as a separate change.